packages/marshaller/src/ddl2/model/element.ts

```diff
@@ -79,8 +79,13 @@ export class Element extends PropertyExt {
     constructor(ec: ElementContainer) {
         super();
         this._elementContainer = ec;
-        vizID++;
-        this._id = `e_${vizID}`;
+        while (true) {
+            vizID++;
+            this._id = `e_${vizID}`;
+            if (!this._elementContainer.elementExists(this._id)) {
+                break;
+            }
+        }
```

Comment:
What could cause duplicate `this._id`? It seems like this should never be necessary?

Reply:
Here is the scenario - create a dashboard with 4 widgets and then delete the first 2 and save it.

Now when you open it the max ID is e_4 - but there are only "2" widgets so the internal counter is now at 2

At some point there is the potential that there will be a collision when the user adds another 2 widgets...